frontend-react/src/components/SideNav.tsx

```diff
@@ -0,0 +1,57 @@
+import { SideNav } from "@trussworks/react-uswds";
```

Comment:
Can we add this file to a new `src/shared` directory for common components?  cc @etanb who's doing a similar thing with the new Table component.